core/src/main/scala/io/finch/Endpoint.scala

```diff
@@ -494,4 +494,20 @@ object Endpoint {
    * Generically derive a very basic instance of [[Endpoint]] for a given type `A`.
    */
   def derive[A]: GenericDerivation[A] = new GenericDerivation[A]
+
+  implicit val applicativeEndpoint: Applicative[Endpoint] = new Applicative[Endpoint] {
+
+    override def pure[A](x: A): Endpoint[A] = new Endpoint[A] {
+      override def apply(input: Input): Result[A] = Some(input -> Eval.now(Future value Ok(x)))
```

Comment:
Yea, no problem.

I prefer this syntax in case if there is a lot of braces. Otherwise it starts looking like a CL dialect :smile: